### ISSUE TITLE
Ability to send raw search

### ISSUE THREAD

**Issue by gjedeer** [funkyg/funkytunes#23]
Some albums are not on discogs but are on TPB/ST.

I thought about adding a menu item below all the search results: "search for [raw search query here]"

Not sure how to explain it so here's a pic: http://f.gdr.name/funkyfunky.png Pressing the extra list item would just search TPB and ST for "gang albanii" and play the first result.

I looked into implementing it but can't wrap my head around the list adapter.

**Comment by funkyg** [comment]
so kind of like "i'm feeling lucky" on google? :D 

the library i used for recyclerview adapter is really basic and doesnt support footer. but there are other libraries with more features, eg this:
https://github.com/h6ah4i/android-advancedrecyclerview

**Comment by gjedeer** [comment]
Yeah, kind of like "im feeling lucky". I just wanted to play stuff that's missing from discogs.

Will look at the view, i'm not great with android gui code but will try
